src/extension.ts

```diff
@@ -7,19 +7,28 @@ const OS: string = platform();
 
 export function activate(context: ExtensionContext) {
 
-  context.subscriptions.push(commands.registerCommand('extension.dash.specific', () => {
-    searchSpecific();
+  context.subscriptions.push(commands.registerCommand('extension.dash.syntax', () => {

```

Comment:
I thought this is what you said was the breaking change

Edit: I shouldn't be writing this at 3 AM. /sigh. Of course. That was the whole reason for that part of the revision.

I went ahead and made sure all the keybinds and changes work